sale_automatic_workflow_ignore_exception/models/automatic_workflow_job.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AutomaticWorkflowJob(models.Model):
+
+    _inherit = "automatic.workflow.job"
+
+    def _do_validate_sale_order(self, sale, domain_filter):
+        if sale.workflow_process_id.ignore_exception_when_confirm:
+            sale = sale.with_context(ignore_exception_when_confirm=True)
```

Comment:
@jcoux I'm wondering if:
```suggestion
            sale.write({"ignore_exception": True})
```
is not better ?

Reply:
It can be more easy.

But in the original module, we have a checked to bypass the sale order validation:
https://github.com/OCA/sale-workflow/blob/50675a1ffb8ac8f60993c22a7246ec321bb0c522/sale_automatic_workflow/models/automatic_workflow_job.py#L43

I don't want to ignore exception in this case.
Then the context option seems to be the best way for me.